java/arcs/core/data/Capabilities.kt

```diff
@@ -15,8 +15,10 @@ package arcs.core.data
 data class Capabilities(val capabilities: Set<Capability>) {
     fun isEmpty(): Boolean = capabilities.isEmpty()
 
-    operator fun contains(other: Capabilities): Boolean =
-        capabilities.containsAll(other.capabilities)
+    operator fun contains(other: Capabilities): Boolean {
+        if (other.isEmpty()) return isEmpty()
```

Comment:
empty capabilities are kind of special, because they only match other explicitly empty capabilities... this will be improved when we have defined all the real capabilities that we need, but for now this serves the short term needs we have. 
let's talk more offline, i'm open to suggestions.